src/sentry.ts

```diff
@@ -0,0 +1,41 @@
+import * as Sentry from '@sentry/node';
+import * as Tracing from '@sentry/tracing';
+import { SamplingContext } from '@sentry/types';
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
+
+import { SENTRY_DSN, SENTRY_ENVIRONMENT } from './config';
+import { HEALTHCHECK_PATH } from './constants';
+import { logger } from './logger';
+
+/**
+ * Creates configuration for sentry.io.
+ */
+export function doSentrySetup(app: core.Express): void {
+    Sentry.init({
+        dsn: SENTRY_DSN,
+        environment: SENTRY_ENVIRONMENT,
+        integrations: [
+            new Sentry.Integrations.Http({
+                tracing: true,
+            }),
+            new Tracing.Integrations.Express({
+                app,
+            }),
+        ],
+        tracesSampler: (context: SamplingContext) => {
+            // Do not submit traces for health check.
+            //
+            if (context?.transactionContext?.name.includes(HEALTHCHECK_PATH)) {
+                return false;
+            }
+
+            return true;
```

Comment:
How expensive tracings are? Would it make sense to set this lower? (say 1%.. or 10%)?